### PR TITLE
Added paragraph spacing for legibility

### PR DESCRIPTION
**Pull request by ron-reaser**
These are the correct changes for adding paragraph spacing. This applies the change both in the base stylesheet and the minified one.

### CODE REVIEW

**Comment by vitusventure** [comment]
I built the new changes out to the test bucket again, but it looks like it is still rendering the same.  Looking at the code, it seems that the bootstrap-custom CSS already applies a 10px bottom margin to p elements, and some quick searching seems to indicate adding a margin to <br> doesn't actually do anything: http://stackoverflow.com/a/7614982.

Are you seeing the expected changes on your own test builds?

**Comment by ron-reaser** [comment]
Yes. The CSS rule I added in is as follows.

`p br {margin-bottom: 5px;}`

It specifically adds spacing after the break tags inside of paragraphs in the content of most pages. This is in addition to Bootstrap's existing margin applied to the bottom of paragraphs themselves. I chose to put the change in the base CSS instead of the Bootstrap CSS since I think it's more fitting there.

If you open a developer console in Firefox or Chrome, you can add that rule and see the effect apply on pages such as [advantage and disadvantage](http://www.5thsrd.org/rules/advantage_and_disadvantage/), which contains paragraphs with breaks in them as a decent example.

There may be a caching issue. You might need to increase the version number to force the stylesheets to be reloaded. Other than that, I don't know what could be the problem.

**Comment by vitusventure** [comment]
I have bumped the version numbers, but I am still not seeing any rendering difference ([screenshot](http://i.imgur.com/cJdKWsP.png)).  Could you take a look at the site in the [test bucket](http://5thsrd.org-test.s3-website-us-east-1.amazonaws.com), and let me know if that is what you are expecting?  

That bucket is populated by pulling down your remote branch, then using the build_to_test.sh script.

**Comment by ron-reaser** [comment]
Yes, I see the change when I view the test bucket but not in your screenshot. However, I just checked and I only see the effect in Firefox, not Chrome, which does sort of make sense because this was a CSS hack.

It would be better to refactor the documents to use a `<p>` tag around each paragraph rather than a `<p>` tag around an entire section with internal `<br />` tags to split them into paragraphs. I mean, the semantics being produced by mkdocs is misguided.

I think that might be due to the Markdown sources themselves not having paragraph gaps between them. If you take, for example, the advantages and disadvantages Markdown and add line separations between the paragraphs, I'm betting that mkdocs will correctly wrap each with a `<p>` tag so that Bootstrap's existing paragraph styling will show a separation between them.

Basically, this CSS fix isn't sufficient and there's no way to make it sufficient, since it's attempting to fix a problem that should be prevented earlier in parsing. So you can close this pull request, since it won't work.

I could open a separate pull request to make a change to the advantages and disadvantages Markdown to test out my conjecture above, and then if that fixes the problem I could probably automate applying the fix to the rest of the files, but that's up to you. It's a pretty big undertaking.

**Comment by vitusventure** [comment]
Gotcha, I get what is going on now.

I have pushed an experimental version of the site to the test bucket above, which should convert everything to use actual <p> tags (through the merciless abuse of sed).  Could you take a look at the test bucket, and let me know if that is what you are expecting/general thoughts?

**Comment by ron-reaser** [comment]
Yeah, looking at the source code of the pages, that looks more semantically correct. It's using all `<p>` tags for paragraphing in their intended way, which has the bonus of working better with the stylesheet. In my opinion, that's the right thing to do, and in that case it doesn't need the CSS edits at all, since Boostrap already had the right paragraph margin-bottoms in place.

Did you just mass-replace all new lines in the Markdown with doubled new lines to get mkdocs to behave correctly?
